src/components/dragDrop/Drag.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import "./dragDrop.css";
+
+// FSBL.addEventListener("onReady", function () {
+export default class Drag extends React.Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            moving: 0
+        }
+    }
+    mouseup = () => {
+        console.log('mouseUp:', this.target)
+        window.removeEventListener('mousemove', this.divMove, true);
+    }
+    mousedown = () => {
+        console.log('mousedown:', this.target)
+        window.addEventListener('mousemove', this.divMove, true);
+    }
+    divMove = (e) => {
+        console.log('divmove:', e.target)
+        var div = document.getElementById(this.state.moving);
+        div.style.position = 'absolute';
+        div.style.top = e.clientY + 'px';
+        div.style.left = e.clientX + 'px';
+    }
+    componentDidMount = () => {
+        console.log('mounting')
+        var div = document.getElementById('0')
+
+        div.addEventListener('mousedown', this.mousedown, true)
+        window.addEventListener('mouseup', this.mouseup, true)
+    }
+    render() {
+        return (
+            <div className="container">
+                <ul>
+                    <li>
+                        <a>
+                            <i className="fa fa-plus" aria-hidden="true" />
```

Comment:
Is this using finsemble's icons or was font awesome installed separately? I haven't seen the mockups so I don't know if this is an issue, but I think all icons should come from our finsemble set `ff-plus`

Reply:
This is font awesome and it was installed separately. Where would I find the icons for finsemble?